rocketpy/plots/flight_plots.py

```diff
@@ -99,19 +97,30 @@ def trajectory_3d(self):
             self.flight.z[:, 1] - self.flight.env.elevation,
             linewidth="2",
         )
-        ax1.scatter(0, 0, 0)
+        ax1.scatter(
+            self.flight.x(0),
+            self.flight.y(0),
+            self.flight.z(0) - self.flight.env.elevation,
+            color="black",
+        )
+        ax1.scatter(
+            self.flight.x(self.flight.t_final),
+            self.flight.y(self.flight.t_final),
+            self.flight.z(self.flight.t_final) - self.flight.env.elevation,
+            color="red",
+            marker="X",
+        )
         ax1.set_xlabel("X - East (m)")
         ax1.set_ylabel("Y - North (m)")
         ax1.set_zlabel("Z - Altitude Above Ground Level (m)")
         ax1.set_title("Flight Trajectory")
-        ax1.set_zlim3d([0, max_z])
+        ax1.set_zlim3d([min_z, max_z])
         ax1.set_ylim3d([min_xy, max_xy])
         ax1.set_xlim3d([min_xy, max_xy])
         ax1.view_init(15, 45)
+        ax1.set_box_aspect(None, zoom=0.95)  # 95% for label adjustment
```

Comment:
Quite a ingenious solution, I did not know you could zoom out.

Reply:
if you zoom out 0.90 instead of 0.95, the problem persists, apparently.

Ingenious is a quite accurate way to describe this solution hahahaha